src/components/theme-icon/theme-icon.tsx

```diff
@@ -23,17 +23,18 @@ const ThemeIcon = ({
   svg,
   colorType,
   size = 'normal',
+  fill,
   ...props
 }: ThemeIconProps): JSX.Element => {
   const {theme, themeName} = useTheme();
 
-  const fill = getFill(theme, themeName, colorType);
+  const fillToUse = fill || getFill(theme, themeName, colorType);
```

Comment:
When using `fill: undefined` on `ThemeIcon` it didn't fallback to the fill based on the theme, but it just used no fill and the icon was not visible.